bee-common/bee-packable-derive/src/attribute/mod.rs

```diff
@@ -1,14 +1,18 @@
 // Copyright 2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
+mod pack_err_with;
```

Comment:
Tbh I prefer consistency to short name, can we use `error` everywhere instead of `err` ?